lib/currency_cloud/payment_confirmation_result.rb

```diff
@@ -0,0 +1,5 @@
+module CurrencyCloud
+  class PaymentConfirmationResult
+    include CurrencyCloud::Resource
+  end
+end
```

Comment:
@cpacalevcc Any reason we are not following the setup followed by other classes

e.g.
resource: <some resource>
actions: <actions allowed>

Reply:
@WorkSutton the resource and actions are declared in payment.rb. Similar setup as it was before, just in other classes.